conformance/tests/Dockerfile

```diff
@@ -9,3 +9,5 @@ COPY --link go.sum /go/src/github.com/nginxinc/nginx-kubernetes-gateway/
 RUN go mod download
 
 COPY --link conformance/tests /go/src/github.com/nginxinc/nginx-kubernetes-gateway/conformance/tests/
+
+CMD go test -timeout 25m -v . -tags conformance -args --gateway-class=${GATEWAY_CLASS} --debug --supported-features=${SUPPORTED_FEATURES} --exempt-features=${EXEMPT_FEATURES}
```

Comment:
You can remove the timeout flag now. Now that the test runner can reach the NKG pods, the tests are running much faster.